airgun/entities/computeresource.py

```diff
@@ -59,6 +61,25 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def list_computeprofiles(self, entity_name, computeprofile_name):
```

Comment:
what does that method do?

Reply:
This method does this:
1. Select the created ec2 CR.
2. Click Compute Profile tab.
3. Select (3-Large) and submit.